pkg/server/service.go

```diff
@@ -26,10 +26,12 @@ import (
 	"time"
 
 	"github.com/containerd/containerd"
+	"github.com/containerd/containerd/log"
 	"github.com/containerd/containerd/plugin"
 	cni "github.com/containerd/go-cni"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
+	"golang.org/x/net/context"
```

Comment:
This pulled in golang.org/x/net/context for some reason.. It's just a type alias for context.Context though so it doesn't really matter. https://pkg.go.dev/golang.org/x/net/context#Context